paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/activity/PaymentSheetPlaygroundActivity.kt

```diff
@@ -159,12 +160,32 @@ class PaymentSheetPlaygroundActivity : AppCompatActivity() {
         super.onCreate(savedInstanceState)
         setContentView(viewBinding.root)
 
-        paymentSheet = PaymentSheet(this, ::onPaymentSheetResult)
+        paymentSheet = PaymentSheet(this, ::onPaymentSheetResult) { paymentMethodId ->
```

Comment:
Nit: Can we add named parameters here? For anyone looking at this as a sample (which is not recommended, but will still happen), it’ll make it more clear what’s happening here.